src/Microsoft.Identity.Client/Cache/Items/MsalRefreshTokenCacheItem.cs

```diff
@@ -37,18 +37,19 @@ internal MsalRefreshTokenCacheItem()
         {
             CredentialType = MsalCacheCommon.RefreshToken;
         }
-        internal MsalRefreshTokenCacheItem(string environment, string clientId, MsalTokenResponse response) :
-            this(environment, clientId, response.RefreshToken, response.ClientInfo)
+        internal MsalRefreshTokenCacheItem(string environment, string clientId, MsalTokenResponse response, string userId=null) :
+            this(environment, clientId, response.RefreshToken, response.ClientInfo, userId)
         {
         }
 
-        internal MsalRefreshTokenCacheItem(string environment, string clientId, string secret, string rawClientInfo) : this()
+        internal MsalRefreshTokenCacheItem(string environment, string clientId, string secret, string rawClientInfo, string userId=null) : this()
         {
             ClientId = clientId;
             Environment = environment;
             Secret = secret;
             RawClientInfo = rawClientInfo;
 
+            HomeAccountId = userId;
```

Comment:
>HomeAccountId = userId; [](start = 12, length = 23)

not sure

Reply:
same comment as for AT and IDT

---
In reply to: [252132159](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/834#discussion_r252132159) [](ancestors = 252132159)